code/modules/mob/living/carbon/human/species_types/oozeling/_oozelings.dm

```diff
@@ -101,14 +108,15 @@
 	if(!wetness)
 		return
 
-	if(wetness.stacks > (DAMAGE_WATER_STACKS))
+	if(wetness.stacks > DAMAGE_WATER_STACKS)
 		slime.blood_volume = max(slime.blood_volume - (2 * seconds_per_tick), 0)
-		if (SPT_PROB(25, seconds_per_tick))
+		slime.balloon_alert(slime, "water is diluting you, you need to dry off!")
```

Comment:
Thats a very long balloon alert.
Honestly idk about a good message to send here. Maybe `too wet, dry off!` or just `too wet!`